airbyte-cdk/python/setup.py

```diff
@@ -46,7 +46,7 @@
     packages=find_packages(exclude=("unit_tests",)),
     package_data={"airbyte_cdk": ["py.typed", "sources/declarative/declarative_component_schema.yaml"]},
     install_requires=[
-        "airbyte-protocol-models==1.0.0",
+        "airbyte-protocol-models==0.3.5",
```

Comment:
This _looks_ like a downgrade... but I don't think we ever were on v1.0.0?

Reply:
Correct.  We were never on 1.0.0 even though we set that as the dependency version.  The new changes were released as a patch on the current version (0.3), which is the reason for the correction.